README.md

```diff
@@ -223,4 +223,128 @@
+| **I-03**: Minimal borrow amount mitigation could be avoided by first depositor                    | Informational   | Acknowledged |
+| **I-04**: Minimal borrow amount mitigation could be somewhat bypassed                             | Informational   | Acknowledged |
+| **I-05**: positionIndexes and positions aren’t deleted when deleting positionTokens               | Informational   | Acknowledged |
+| **I-06**: TakerActions.sol TakerLocalVars.factor is unused                                        | Informational   | Fixed        |
+| **I-07**: Use string.concat() or bytes.concat() instead of abi.encodePacked                       | Informational   | Acknowledged |
+| **I-08**: Default Visibility for constants                                                        | Informational   | Acknowledged |
+| **I-09**: Event is never emitted                                                                  | Informational   | Acknowledged |
+| **I-10**: Change uint to uint256                                                                  | Informational   | Acknowledged |
+| **Gas-01**: State variables only set in the constructor should be declared immutable              | Informational   | Acknowledged |
+| **Gas-02**: uint256 to bool mapping: Utilizing Bitmaps to save on Gas                             | Informational   | Acknowledged |
+| **Gas-03**: Increments/decrements can be unchecked in for-loops                                   | Informational   | Acknowledged |
+| **Gas-04**: ++i costs less gas compared to i++ or i += 1                                          | Informational   | Acknowledged |
+
+## 10. Sherlock 3rd contest (October 13, 2023 - October 27, 2023)
+#### Contest: https://audits.sherlock.xyz/contests/114
+#### Report: [Sherlock 3 Report](https://audits.sherlock.xyz/contests/114/report) ([Local](./sherlock/Contest3.md))
+#### Scope: [Protocol](https://github.com/ajna-finance/ajna-core) Contracts
+#### Findings: 1 High, 4 Medium
+
+# Findings Summary
+
+| Findings                                                                                             | Severity        | Resolution   |
+|------------------------------------------------------------------------------------------------------|-----------------|--------------|
+| **H-1**: Reserves can be stolen by settling artificially created bad debt from them                  | High            | Fixed        |
+| **M-1**: Wrong auctionPrice used in calculating BPF to determine bond reward (or penalty)            | Medium          | Fixed        |
+| **M-2**: Incorrect implementation of BPF leads to kicker losing rewards in a take action             | Medium          | Fixed        |
+| **M-3**: HPB may be incorrectly bankrupt due to use of unscaled value in _forgiveBadDebt             | Medium          | Fixed        |
+| **M-4**: First pool borrower pays extra interest                                                     | Medium          | Disputed     |
```

Comment:
This was fixed, not disputed: https://github.com/sherlock-audit/2023-09-ajna-judging/issues/26

Reply:
Correct, good catch. Thanks -> https://github.com/ajna-finance/audits/pull/3/commits/4799867b5d043fe1f3bf947e339ae60d6a3694d2